collector.go

```diff
@@ -30,11 +30,12 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-func CollectorInitial(target string, protocol int, duration time.Duration) Collector {
+func CollectorInitial(target string, protocol int, duration time.Duration, highDrift time.Duration) Collector {
```

Comment:
```suggestion
func CollectorInitial(target string, protocol int, duration, highDrift time.Duration) Collector {
```

to make the linter happy

Reply:
Corrected